Source/Diagnostics/ReducedDiags/LoadBalanceCosts.cpp

```diff
@@ -49,8 +49,9 @@ void LoadBalanceCosts::ComputeDiags (int step)
     m_nBoxesMax = std::max(m_nBoxesMax, nBoxes);
 
     // resize and clear data array
-    m_data.resize(m_nDataFields*nBoxes, 0.0);
-    m_data.assign(m_nDataFields*nBoxes, 0.0);
+    const size_t m_dataSize = m_nDataFields*nBoxes;
```

Comment:
this is a local variable, not a member variable :)
```suggestion
    const size_t dataSize = m_nDataFields*nBoxes;
```